blueprints/policies/feature-rollout/base/policy.libsonnet

```diff
@@ -183,11 +180,10 @@ function(cfg, metadata={}) {
 
   local addEMALatencyDriver = function(driverAccumulator, driver) {
     local flux_meter_name = policyName + '/ema_latency/' + std.toString(driverAccumulator.ema_latency_driver_count),
-    local evaluationInterval = params.policy.evaluation_interval,
     local latencySignalName = 'LATENCY_' + std.toString(driverAccumulator.ema_latency_driver_count),
     local q = 'sum(rate(flux_meter_sum{flow_status="OK", flux_meter_name="%(flux_meter_name)s"}[5s]))/sum(rate(flux_meter_count{flow_status="OK", flux_meter_name="%(flux_meter_name)s"}[5s]))' % { flux_meter_name: flux_meter_name },
```

Comment:
The current 5s interval is not enough with scrape of 5s. Change it 15s

Please change all 5s and 10s intervals in the queries to 15s